### PR TITLE
Include registry data in DR snapshots

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: addons/kotsadm/alpha/tmpl-secret-s3.yaml [hunk 1]
```diff
@@ -2,9 +2,6 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: kotsadm-s3
-  labels:
```
Comment (salahalsaleh) [inline addons/kotsadm/alpha/tmpl-secret-s3.yaml:5]: we should remove these in the install script explicitly too

Comment (salahalsaleh) [reply addons/kotsadm/alpha/tmpl-secret-s3.yaml:5]: unless we always apply the secret

File: addons/registry/2.7.1/kustomization.yaml [hunk 2]
```diff
@@ -1,4 +1,3 @@
 namespace: kurl
 
 resources:
```
Comment (salahalsaleh) [inline addons/registry/2.7.1/kustomization.yaml:3]: is this key still needed?

File: scripts/install.sh [hunk 3]
```diff
@@ -379,6 +379,7 @@ function main() {
     addon_for_each addon_load
     init
     apply_installer_crd
+    create_registry_service
```
Comment (areed) [inline scripts/install.sh:382]: won't be defined when registry add-on isn't included in your spec

File: addons/registry/2.7.1/kustomization.yaml (hunk 2 above)
Comment (divolgin) [reply addons/registry/2.7.1/kustomization.yaml:3]: not sure... this is how I tested :)

**Review by areed — COMMENTED** [review]
I don't understand why the KURL_REGISTRY_IP is an option for the user and why it's a bash flag instead of part of the yaml spec.

**Comment by divolgin** [comment]
@areed KURL_REGISTRY_IP is not something that can ever be defined by a vendor, and it's really only applicable when restoring another kURL cluster.  So I made it a command line flag only, at least for now.

**Review by areed — APPROVED** [review]

**Pull request merged**